transaction_integration_test.go

```diff
@@ -725,3 +726,185 @@ func TestTransactionCreateWithCustomFields(t *testing.T) {
 		t.Fatalf("Returned custom fields doesn't match input, got %q, want %q", x, customFields)
 	}
 }
+
+func TestHoldInEscrowOnCreate(t *testing.T) {
+	testSubMerchantAccountId := getSubMerchantAccount(t)
```

Comment:
Can we mark all these new tests as parallel, by adding `t.Parallel()` as the first line of each function? We run all the tests across 15 goroutines since the majority are integration tests and spend most of their time blocked on network calls.